src/lineup/internal/scorecmds.ts

```diff
@@ -70,11 +71,11 @@ export async function pushScoreAsync(graph: ProvenanceGraph, provider: IObjectRe
   const currentParams = {id: scoreId, params, storedParams};
   const result = await addScoreAsync([provider], currentParams);
   const toStoreParams = {id: scoreId, params: storedParams};
-  return graph.pushWithResult(action(meta(`Add ${scoreName}`, cat.data, op.create), CMD_ADD_SCORE, addScoreImpl, [provider], toStoreParams), result);
+  return graph.pushWithResult(action(meta(`${i18n.t('tdp:core.lineup.scorecmds.add')} ${scoreName}`, cat.data, op.create), CMD_ADD_SCORE, addScoreImpl, [provider], toStoreParams), result);
```

Comment:
```suggestion
  return graph.pushWithResult(action(meta(i18n.t('tdp:core.lineup.scorecmds.add', {scoreName}), cat.data, op.create), CMD_ADD_SCORE, addScoreImpl, [provider], toStoreParams), result);
```

Update language string (see above).

Reply:
Thank you for the feedback.